cfgov/unprocessed/js/modules/TabTrigger.js

```diff
@@ -0,0 +1,74 @@
+// Required modules.
+const EventObserver = require( '../modules/util/EventObserver' );
+
+// Key code for the tab key on the keyboard.
+const KEY_TAB = 9;
+
+/**
+ * Dynamically creates an HTML element and attaches it to a DOM node.
+ * @param  {string} type - The type of HTML node to create.
+ * @param  {HTMLNode} target - An HTML element to insert the new node into.
+ * @returns {HTMLNode} The newly created HTML node.
+ */
+function createElement( type, target ) {
+  const elem = document.createElement( type );
+  target.appendChild( elem );
+  return elem;
+}
+
+/**
+ * MegaMenu
+ * @class
+ *
+ * @classdesc Initializes a new MegaMenu organism.
+ *
+ * @param {HTMLNode} element
+ *   The DOM element within which to search for the organism.
+ * @returns {TabTrigger} An instance.
+ */
+function TabTrigger( element ) {
+
+  const _that = this;
```

Comment:
make this `_this`

Reply:
Changed, though note `_that` is used in the expandables. Probably should be updated.